resolvers/ambulance.js

```diff
@@ -54,8 +54,8 @@ const ambulanceResolvers = {
       });
       return db.ambulance.findByPk(args.id);
     },
-    deleteAmbulance: async (parent, args) => {
-      await db.patient
+    deleteAmbulance: async (parent, args) =>
+      db.patient
```

Comment:
i'm not a fan of promise chaining especially this many times - i know i've leaned towards it before but i feel like there's been issues with readability and swallowing errors thrown inside promises; to be clear, i think one promise + one chain is reasonable, but beyond that it gets kinda messy, especially if there's not a whole lot happening inside them

i'm liking `async`/`await` more now, as it's more intuitive (maybe just for me) and easier to deal with errors - i've been  pushing for that with new devs (e.g. https://github.com/uwblueprint/paramedics-web/pull/66)

just wanted to thoughts from you and @aaronabraham311 on what pattern we should move forward with

Reply:
I personally think it's cleaner to use promise chaining here if we're using the result returned by these promises. Since like I mentioned, `try` and `catch` wouldn't do anything for us because `destroy` always results in a fulfilled promise, but it's just a matter of whether its return value is `0` or `1`.

Reply:
I don't think it's about whether you consistently use async/await or promise chaining, since they each have their uses https://medium.com/better-programming/should-i-use-promises-or-async-await-126ab5c98789

Reply:
Thanks for the link, that does help with clarifying between the two options - in that case I'm fine with keeping this the way it is